2023/zhCN/API6:2023 服务器端请求伪造.md

```diff
@@ -0,0 +1,136 @@
+# API6:2023 服务器端请求伪造
+
+| 威胁因素/攻击向量                                            | 安全弱点                                                     | 影响                                                         |
+| ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| API特定：可利用性**2**                                       | 普遍性**2**：可检测性**1**                                   | 技术性**2**：业务特定                                        |
```

Comment:
Compared with the original version. I think from Chinese best-practice translation 
```
Threat agents/Attack vectors --> 威胁因素/攻击向量 --> 威胁源/攻击特征

API Specific --> API特定 -->API相关
Business Sepcific --> 业务特定 --> 业务相关
 ```

Reply:
done